package.json

```diff
@@ -18,7 +18,10 @@
   ],
   "dependencies": {
     "bull": "^4.15.1",
-    "bullmq": "^5.8.3",
+    "bullmq": "^5.31.1",
+    "bullmq-v3": "npm:bullmq@^3.16.2",
+    "bullmq-v4": "npm:bullmq@^4.18.2",
+    "bullmq-v5": "npm:bullmq@^5.31.1",
```

Comment:
@manast Shouldn't these be optional `peerDependencies`? As in:

``` json
{
  "peerDependencies": {
    "bullmq-v3": "npm:bullmq@^3.16.2",
    "bullmq-v4": "npm:bullmq@^4.18.2",
    "bullmq-v5": "npm:bullmq@^5.31.1
  },
  "peerDependenciesMeta": {
    "bullmq-v3": {
      "optional: true
    },
    "bullmq-v4": {
      "optional: true
    },
    "bullmq-v5": {
      "optional: true
    }
  }
}
```

This way, we don't have to install all versions of BullMQ and their dependencies (including outdated and potentially vulnerable ones).

Reply:
yes, but it will be difficult leaving to the user to manually know which dependencies to install, specially using the CLI version.

Reply:
We are taking care of vulnerabilities in this package as a whole anyway btw.